### PR TITLE
Deprecate waas/v2-alpha versions

### PR DESCRIPTION
**Pull request by Uzlopak**
As planned we deprecate waas/v2-alpha versions

### CODE REVIEW

File: src/validateVersion.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  'waas/v1alpha4',
+  'waas/v1beta1',
+  'waas/v2'
+]
+
+export function validateVersion (version: string) {
+  if (
+    validVersions.includes(version) === false
+  ) {
+    throw new Error(`The version ${version} is deprecated and is decommissioned on 01.03.2023. Please update the setup-tools-for-waas action to use version waas/v2(latest) instead.`)
```
Comment (uwehdaub) [inline src/validateVersion.ts]: I don't know, whether I like to refer to `waas/latest` ...
At least the current spelling is misleading.
We should replace it with 
_... use version `waas/v2` or `waas/latest` instead._

**Review by uwehdaub — APPROVED** [review]
LGTM

File: src/validateVersion.ts (hunk 1 above)
Comment (Uzlopak) [reply src/validateVersion.ts]:
```suggestion
    throw new Error(`The version ${version} is deprecated and is decommissioned on 01.03.2023. Please update the setup-tools-for-waas action to use version waas/v2 or waas/latest instead.`)
```

**Review by Navya-Teja-Otturu — APPROVED** [review]
Looks good to me.

**Pull request merged**